src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo;
+  oncloseButtonClick: (value: number) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
```

Comment:
check `network` tab in browser when `TodoModal` is opened
there is a lot of api requests, check how it works and fix it